content/en/docs/setup/production-environment/container-runtimes.md

```diff
@@ -210,7 +210,7 @@ sudo yum update -y && sudo yum install -y containerd.io
 ```shell
 ## Configure containerd
 sudo mkdir -p /etc/containerd
-sudo containerd config default > /etc/containerd/config.toml
+sudo containerd config default | sudo tee /etc/containerd/config.toml
```

Comment:
do we need to run 'conteinerd config default' with sudo?

Reply:
I can run this command without sudo.

Reply:
`containerd config default` works with and without sudo when used along with `sudo tee`. I've made this change to be in line with what we have been using in the documentation so far(containerd config default, with sudo). Here are a few other cases
```
[worker@a-worker ~]$ containerd config default > /etc/containerd/config.toml
-bash: /etc/containerd/config.toml: Permission denied
[worker@a-worker ~]$ sudo containerd config default > /etc/containerd/config.toml
-bash: /etc/containerd/config.toml: Permission denied
[worker@a-worker ~]$ containerd config default | sudo tee /etc/containerd/config.toml
```

Reply:
In many other places of the same doc, we are not forcing users to run 'sudo containerd config default', so we'd better be consistent?

Reply:
So below is the current state of `sudo containerd config default` (9 occurrences) vs `containerd config default`( 5 occurrences) in the master branch. I went with what is being used in the majority of cases.  Since both with and without sudo works, I'm fine with using either of these as long as we are consistent across all occurrences. I don't have a preference either way. What are your thoughts?

```
content/en/docs/setup/production-environment/container-runtimes.md:sudo containerd config default | sudo tee /etc/containerd/config.toml
content/en/docs/setup/production-environment/container-runtimes.md:sudo containerd config default | sudo tee /etc/containerd/config.toml
content/en/docs/setup/production-environment/container-runtimes.md:containerd config default | sudo tee /etc/containerd/config.toml
content/en/docs/setup/production-environment/container-runtimes.md:sudo containerd config default > /etc/containerd/config.toml
content/id/docs/setup/production-environment/container-runtimes.md:containerd config default > /etc/containerd/config.toml
content/id/docs/setup/production-environment/container-runtimes.md:containerd config default > /etc/containerd/config.toml
content/ja/docs/setup/production-environment/container-runtimes.md:containerd config default > /etc/containerd/config.toml
content/ja/docs/setup/production-environment/container-runtimes.md:containerd config default > /etc/containerd/config.toml
content/ko/docs/setup/production-environment/container-runtimes.md:sudo containerd config default | sudo tee /etc/containerd/config.toml
content/ko/docs/setup/production-environment/container-runtimes.md:sudo containerd config default > /etc/containerd/config.toml
content/zh/docs/setup/production-environment/container-runtimes.md:sudo containerd config default > /etc/containerd/config.toml
content/zh/docs/setup/production-environment/container-runtimes.md:sudo containerd config default > /etc/containerd/config.toml
content/zh/docs/setup/production-environment/container-runtimes.md:sudo containerd config default > /etc/containerd/config.toml
content/zh/docs/setup/production-environment/container-runtimes.md:sudo containerd config default > /etc/containerd/config.toml
```

Reply:
ok, let's merge it as it is. I'll get rid of sudo in a separate PR.

Reply:
@bart0sh unless you have already started working on this, I can submit a separate PR without sudo in the command as soon as this gets merged. I already have context and a list to make these changes.